lang/en/bigbluebuttonbn.php

```diff
@@ -351,11 +351,11 @@
 $string['view_recording_list_recording'] = 'Recording';
 $string['view_recording_button_import'] = 'Import recording links';
 $string['view_recording_button_return'] = 'Go back';
-$string['view_recording_format_podcast'] = 'podcast';
-$string['view_recording_format_presentation'] = 'presentation';
-$string['view_recording_format_screenshare'] = 'screenshare';
-$string['view_recording_format_statistics'] = 'statistics';
-$string['view_recording_format_video'] = 'video';
+$string['view_recording_format_podcast'] = 'Podcast';
+$string['view_recording_format_presentation'] = 'Presentation';
+$string['view_recording_format_screenshare'] = 'Screenshare';
+$string['view_recording_format_statistics'] = 'Statistics';
+$string['view_recording_format_video'] = 'Video';
```

Comment:
We are still missing notes=Notes. The fallback will do it normally, but no for non-English sites

Reply:
Added it to the language string.